syntax = "proto3";

package packet;

message FramePacket {
    int32 rows = 1;
    int32 cols = 2;
    int32 elt_type = 3;
    int32 elt_sizea = 4;
    int32 elt_sizeb = 7;
    int32 rowsb = 8;
    int32 colsb = 9;
    bytes mat_dataA = 5;
    bytes mat_dataB = 6;
}